### DISCUSSION TITLE
Score vs Compose Bridge

### DISCUSSION THREAD

**Discussion by fmontorsi-equinix** [score-spec/spec#111] [General]
Hey all,
I'm totally a newbie just interested into Score. I actually plan to start using it in the next few days/weeks for some project.
However just a couple days back I stumbled on this project: https://docs.docker.com/compose/bridge/

I guess Compose Bridge is a new experimental project born after Score. Still I wonder if you can see some obvious difference between the 2 projects because at first glance both Score and Compose Bridge look very very similar, isn't it?

thanks!

**Comment by mathieu-benoit** [comment]
Hi @fmontorsi-equinix , thanks for reaching out and for the question.

I just have a look at this new project from Docker. Interesting but it's just generating Kubernetes manifests from existing Compose files.

Here is top of mind features covered by Score and not by this Bridge project, as far as I can see:
- From a Score file, `score-compose` generates complex Docker Compose files;
- From a Score file, `score-k8s` generates complex Kubernetes manifests files;
- More implementations are coming;
- The major differentiation of Score is the resources dependencies and placeholders resolution.

Let us know if you need any complementary information.

And please, feel free to join the community to ask questions, provide feedback, etc.: https://score.dev/community/.
